test/DataSet.test.js

```diff
@@ -282,7 +285,111 @@ describe('DataSet', function () {
       {id: 3, content: 'Item 3'},
       {id: 4, content: 'Item 4'}
     ]);
+  });
+
+  describe('add', function () {
+    var dataset = new DataSet([]);
```

Comment:
See comments in previous `describe`-blocks on usage of bare data. Please just ensure that data is re-initialized upon execution of every `it`-block.